### ISSUE TITLE
Loss Explosion Issue When Training Model from Scratch

### ISSUE THREAD

**Issue by my-yy** [SWivid/F5-TTS#386]
### Checks

- [X] This template is only for usage issues encountered.
- [X] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [X] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [X] I confirm that I am using English to submit this report in order to facilitate communication.

### Environment Details

ubuntu 22.0.4
torch  2.3.1
8 x H800 80GB

### Steps to Reproduce

Hello, thanks for the inspiring project you've shared. 
I have attempted to train a small model from scratch using the WeNet4TTS Premium dataset.
The hyperparameters are as outlined in your paper:

```python
model_cls = DiT
model_cfg = dict(dim=768, depth=18, heads=12, ff_mult=2, text_dim=512, conv_layers=4)
```

Additionally, I have set the `epochs=1000` , as I observed that the default `epochs=11` resulted in the learning rate decreasing too rapidly.

However, I've encountered the loss begins to grow and eventually explodes during training. 

<img width="1031" alt="screenshot" src="https://github.com/user-attachments/assets/c03636d9-c44a-4ab0-93dc-9b565f25585a">

Have you experienced this issue? I would greatly appreciate any advice or suggestions on how to address this problem.

Thank you in advance for your time and assistance.

Best regards!

**Comment by SWivid** [comment]
We haven't encountered this in our experiments (we trained small models with half the batchsize as base model, and only 800k updates)

**Comment by Standchen** [comment]
<img width="1249" alt="Image" src="https://github.com/user-attachments/assets/9035d935-f2dc-4bf6-a4e1-c559d71c57d9" />

<img width="1248" alt="Image" src="https://github.com/user-attachments/assets/9bc93cb2-3c08-4b12-a935-c5faa0e7bdad" />


Hello,

I've encountered the same issue during multiple training runs. The loss tends to explode at random points, and I can't figure out why this happens.

Interestingly, this issue seems to occur sporadically—sometimes, if I restart the training, it completes successfully without any errors.

Have you managed to solve this issue? If so, I’d greatly appreciate it if you could share your solution.

For reference, here are my training settings:
(I used batch size type `sample` and gradient accumulation.)
```
batch_size_per_gpu: 16
batch_size_type: sample
grad_accumulation_steps: 4
```

Thank you!
